Bug: building v.0.3.2 release

Here is the cli output:

```shell

Run if [ "$RUNNER_OS" = "Linux" ]; then
     _        _   _                 _
 ___| |_ __ _| |_(_) ___      _ __ | |__  _ __
/ __| __/ _` | __| |/ __|____| '_ \| '_ \| '_ \
\__ \ || (_| | |_| | (_|_____| |_) | | | | |_) |
|___/\__\__,_|\__|_|\___|    | .__/|_| |_| .__/   v2.8.3
                             |_|         |_|
Checking if current OS are supported ... Linux x86_64 ubuntu, supported
Checking if necessary tools are installed ... re2c, autopoint not installed on your system
Automatically fixing install-linux-tools ...
[17:14:47] [I] [EXEC] apt-get install -y re2c autopoint
Reading package lists...
Building dependency tree...
Reading state information...
The following NEW packages will be installed:
  autopoint re2c
0 upgraded, 2 newly installed, 0 to remove and 29 not upgraded.
Need to get 878 kB of archives.
After this operation, 3979 kB of additional disk space will be used.
Get:1 file:/etc/apt/apt-mirrors.txt Mirrorlist [144 B]
Get:2 http://azure.archive.ubuntu.com/ubuntu noble/main amd64 autopoint all 0.21-14ubuntu2 [422 kB]
Get:3 http://azure.archive.ubuntu.com/ubuntu noble/main amd64 re2c amd64 3.1-1build1 [456 kB]
Fetched 878 kB in 0s (5422 kB/s)
Selecting previously unselected package autopoint.
(Reading database ... 
(Reading database ... 5%
(Reading database ... 10%
(Reading database ... 15%
(Reading database ... 20%
(Reading database ... 25%
(Reading database ... 30%
(Reading database ... 35%
(Reading database ... 40%
(Reading database ... 45%
(Reading database ... 50%
(Reading database ... 55%
(Reading database ... 60%
(Reading database ... 65%
(Reading database ... 70%
(Reading database ... 75%
(Reading database ... 80%
(Reading database ... 85%
(Reading database ... 90%
(Reading database ... 95%
(Reading database ... 100%
(Reading database ... 220446 files and directories currently installed.)
Preparing to unpack .../autopoint_0.21-14ubuntu2_all.deb ...
Unpacking autopoint (0.21-14ubuntu2) ...
Selecting previously unselected package re2c.
Preparing to unpack .../re2c_3.1-1build1_amd64.deb ...
Unpacking re2c (3.1-1build1) ...
Setting up autopoint (0.21-14ubuntu2) ...
Setting up re2c (3.1-1build1) ...
Processing triggers for man-db (2.12.0-4build2) ...
Not building database; man-db/auto-update is not 'true'.

Running kernel seems to be up-to-date.

No services need to be restarted.

No containers need to be restarted.

No user sessions are running outdated binaries.

No VM guests are running outdated hypervisor (qemu) binaries on this host.
Fix applied successfully!
Checking if musl-wrapper is installed ... musl-wrapper is not installed on your system
Automatically fixing fix-musl-wrapper ...
[17:14:53] [W] Current user is not root, using sudo for running command
[17:14:53] [I] Downloading https://musl.libc.org/releases/musl-1.2.5.tar.gz
[17:15:36] [I] Extracting musl-1.2.5 source to /home/runner/work/clonio-cli/clonio-cli/source/musl-1.2.5 ...
patching file src/locale/iconv.c
Hunk #1 succeeded at 495 (offset -7 lines).
patching file src/locale/iconv.c
Hunk #1 succeeded at 538 (offset -7 lines).
[17:15:36] [I] Installing musl wrapper
[17:15:36] [I] [EXEC] CC=gcc CXX=g++ AR=ar LD=ld ./configure --disable-gcc-wrapper
[17:15:38] [I] [EXEC] CC=gcc CXX=g++ AR=ar LD=ld make -j
[17:15:55] [I] [EXEC] CC=gcc CXX=g++ AR=ar LD=ld sudo make install
Fix applied successfully!
Checking if pkg-config is installed or built ... pkg-config is not installed
Automatically fixing install-pkgconfig ...
[17:15:58] [I] Extracting pkg-config-x86_64-linux package to /home/runner/work/clonio-cli/clonio-cli/pkgroot/x86_64-linux/pkg-config-x86_64-linux ...
Fix applied successfully!
Checking if musl-cross-make is installed ... musl-cross-make is not installed on your system
Automatically fixing fix-musl-cross-make ...
[17:15:58] [W] Current user is not root, using sudo for running command
[17:15:58] [I] Downloading package musl-toolchain-x86_64-linux
curl: (22) The requested URL returned error: 500
[17:16:03] [W] Download package musl-toolchain-x86_64-linux failed: Download failed: Direct command run failed with code: 22
[17:16:03] [W] No alternative package found for musl-toolchain-x86_64-linux, using default mirror.
curl: (22) The requested URL returned error: 500
Fix failed: Download failed: Direct command run failed with code: 22
Failed to apply fix!
Error: Process completed with exit code 1.
```

Link: https://github.com/clonio-dev/clonio-cli/actions/runs/23954918978/job/69871125547#step:4:108

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -91,7 +91,7 @@ jobs:
 
       - name: Download SPC binary
         run: |
-          curl -fsSL "https://github.com/crazywhalecc/static-php-cli/releases/latest/download/${{ matrix.spc-binary }}.tar.gz" \
+          curl -fsSL "https://github.com/crazywhalecc/static-php-cli/releases/download/2.8.4/${{ matrix.spc-binary }}.tar.gz" \
             -o spc.tar.gz
           tar -xzf spc.tar.gz
           chmod +x ./spc

